src/esperanza/finalizationstate.cpp

```diff
@@ -146,10 +155,10 @@ void FinalizationState::InstaJustify() {
   m_last_justified_epoch = m_current_epoch - 1;
 
   if (m_current_epoch > 1) {
-    uint32_t expected_finalized = m_current_epoch - 2;
-    if (GetCheckpoint(expected_finalized).m_is_justified) {
-      GetCheckpoint(expected_finalized).m_is_finalized = true;
-      m_last_finalized_epoch = expected_finalized;
+    uint32_t prev_justified = m_current_epoch - 2;
```

Comment:
I'd rather call this `to_be_finalized` or something similar, cause it might be confusing since we have 2 different epoch that are justified here.

Reply:
I don't have a strong opinion here so as the original name was confusing, changing to `to_be_finalized `